src/java/fr/paris/lutece/plugins/forms/export/AbstractFileGenerator.java

```diff
@@ -73,21 +73,25 @@ public abstract class AbstractFileGenerator implements IFileGenerator
 
     private List<FormExportConfig> _configList = null;
     private Form _form;
+    protected HttpServletRequest _request;
+
 
     /**
      * Constructor
      * 
      * @param _listFormResponseItems
      */
     protected AbstractFileGenerator( String fileName, FormPanel formPanel, List<IFormColumn> listFormColumn, List<FormFilter> listFormFilter,
-            FormItemSortConfig sortConfig, String fileDescription )
+            FormItemSortConfig sortConfig, String fileDescription, HttpServletRequest request, Form form )
```

Comment:
what's the use of HttpServletRequest parameter (it's better to avoid passing this general object as parameter, because it brings a dependency to the HttpServletRequest class, prefer more precise objects)